python_modules/dagster/dagster/check/__init__.py

```diff
@@ -287,6 +287,12 @@ def _check_list_items(obj_list, of_type):
 
 
 def opt_list_param(obj_list, param_name, of_type=None):
+    """Ensures argument obj_list is a list or None; in the latter case, instantiates an empty list
```

Comment:
minor nit. we use single quotes for everything unless you are embedding the double quote in the string itself

Reply:
cool, updated to fix this